data-products/src/test/scala/org/sunbird/analytics/sourcing/TestSourcingSummaryReport.scala

```diff
@@ -0,0 +1,148 @@
+    implicit val jobConfig = JSONUtils.deserialize[JobConfig](config)
+
+    implicit val sqlContext = new SQLContext(sc)
+
+    //mocking for DruidDataFetcher
+    import scala.concurrent.ExecutionContext.Implicits.global
+    val json: String =
+      """
+        |{
+        |    "primaryCategory": "Practise Question Set",
+        |    "createdBy": "0124698765480987654",
+        |    "count": 2.0
+        |  }
+      """.stripMargin
+
+    val doc: Json = parse(json).getOrElse(Json.Null)
+    val results = List(DruidResult.apply(Some(ZonedDateTime.of(2020, 1, 23, 17, 10, 3, 0, ZoneOffset.UTC)), doc))
+    val druidResponse = DruidResponseTimeseriesImpl.apply(results, QueryType.GroupBy)
+
+    implicit val mockDruidConfig = DruidConfig.DefaultConfig
+    val mockDruidClient = mock[DruidClient]
+    (mockDruidClient.doQuery[DruidResponse](_: DruidQuery)(_: DruidConfig)).expects(*, mockDruidConfig).returns(Future(druidResponse)).anyNumberOfTimes()
+    (mockFc.getDruidRollUpClient _).expects().returns(mockDruidClient).anyNumberOfTimes()
+
+    SourcingSummaryReport.execute()
+
+    val report = sqlContext.sparkSession.read
+      .option("header","false")
+      .json("sourcing/SourcingReport.json")
+
+    report.first().getString(0) should be ("0124698765480987654")
```

Comment:
Good to have assertion for all the reports fields. Please add an assertion to the rest of the columns.